src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -481,9 +503,7 @@ internal class PermissionTest : DatabaseTest() {
         updateProject = true,
     )
 
-    permissions.expect(
-        deleteSelf = true,
-    )
+    permissions.expect(deleteSelf = true, readSubmission = true)
```

Comment:
Trailing comma would be good here just to keep future diffs cleaner (but as noted below, this should probably be a separate `expect()` call anyway.)